chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/SurveillanceManagerImpl.java

```diff
@@ -410,21 +410,7 @@ private Surveillance convertToDomain(final PendingSurveillanceEntity pr) {
         surv.setStartDate(pr.getStartDate());
         surv.setEndDate(pr.getEndDate());
         surv.setRandomizedSitesUsed(pr.getNumRandomizedSites());
-        if (pr.getCertifiedProduct() != null) {
```

Comment:
There is some data about the Listing that's needed in the pending surveillance. Database ID, CHPL Product Number, maybe something else. Is that still getting populated somewhere?